invenio_marc21/serializers/dojson.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Base class for dojson based serializers."""
+
+from __future__ import absolute_import, print_function
+
+from dojson.contrib.marc21.utils import GroupableOrderedDict
+
+from invenio_records_rest.serializers.base import PreprocessorMixin
+
+
+class DoJSONSerializer(PreprocessorMixin):
```

Comment:
Hmm, I'm not sure if we should have some doJSON, generic code in this package (because if we will, in the future, create another package extending doJSON with different standard, `DC`, `EAD` then it will be missing this function).
@lnielsen , WDYT ?